README.md

```diff
@@ -49,7 +67,15 @@ function(someHTTPRequest) {
   });
 }
 ```
-As of today there is no support for directly piping of the request stream into a consumer. The files are first written to disk using `os.tmpdir()`. When the consumer stream drained the request stream, files will be automatically removed, otherwise the host OS should take care of the cleaning process.
+
+As of today there is no support for directly piping of the request stream into a consumer when using the standard API. The files are first written to disk using `os.tmpdir()`. When the consumer stream drained the request stream, files will be automatically removed, otherwise the host OS should take care of the cleaning process.
```

Comment:
Hi @christensson, sorry for the delayed answer. 
It all looks good, but I would like to tweak the readme. Essentially what you PR does IS offering a way to directly manipulate the streams so this paragraph doesn't make much sesen anymore. How about instead having a whole paragraph titled "Custome onFile handler"